pkg/provider/azure_loadbalancer_backendpool.go

```diff
@@ -443,22 +444,20 @@ func (bi *backendPoolTypeNodeIP) EnsureHostsInPool(service *v1.Service, nodes []
 			}
 
 			if !existingIPs.Has(privateIP) {
-				name := node.Name
-				if utilnet.IsIPv6String(privateIP) {
-					name = fmt.Sprintf("%s-%s", name, v6Suffix)
-				}
-
-				klog.V(6).Infof("bi.EnsureHostsInPool: adding %s with ip address %s", name, privateIP)
-				*backendPool.LoadBalancerBackendAddresses = append(*backendPool.LoadBalancerBackendAddresses, network.LoadBalancerBackendAddress{
-					Name: pointer.String(name),
-					LoadBalancerBackendAddressPropertiesFormat: &network.LoadBalancerBackendAddressPropertiesFormat{
-						IPAddress: pointer.String(privateIP),
-					},
-				})
+				// name := node.Name
+				// klog.V(6).Infof("bi.EnsureHostsInPool: adding %s with ip address %s", name, privateIP)
```

Comment:
could you keep the debug log for troubleshooting?

Reply:
done